assets/src/dashboard/style.css

```diff
@@ -42,6 +42,7 @@ body.web-story_page_stories-dashboard #wpwrap,
 body.web-story_page_stories-dashboard #wpbody-content {
   position: absolute;
   overflow: hidden;
+  overflow-y: scroll;
```

Comment:
My only worry with this approach is now on the story and template gallery pages we have 2 overflow-y: scroll containers stacked on top of each other. This shouldn't cause any weird scrolling behavior right now because the `Layout.Scrollable` is explicitly set to take up the full size of it's parent, but kind of leads an error prone structure in the future

Reply:
I think using the Layout Provider makes this moot -I'll remove this overflow-y scroll